src/Character.hpp

```diff
@@ -56,6 +57,7 @@ namespace SokuLib
 		CHARACTER_SATORI
 	};
 
+    //! @brief Contains all the information of a character available at character select.
```

Comment:
It's actually updated only after character select. These are used by the loading thread to figure out what to load.